### DISCUSSION TITLE
Launch application without admin privilege

### DISCUSSION THREAD

**Discussion by pbasithmpy** [wixtoolset/issues#7848] [Questions]
I am using wix toolset version 3.11.
I need to launch my application after installation is completed. currently I am using "WixShellExecTarget" for launching application. But it launches with elevated permission. I need to launch the application with user permission only. 

please check the below code,

```
<Property Id="WixShellExecTarget" Value="[#NotifyIconExecutable]" />
        <CustomAction Id="LaunchApplication"
            BinaryKey="WixCA"
            DllEntry="WixShellExec"
            Impersonate="yes" />

        <InstallExecuteSequence>
            <Custom Action="LaunchApplication" After="InstallFinalize">NOT Installed</Custom>
        </InstallExecuteSequence>
```

**note**: I am using Bundle project for combining multiple msi packages to single exe package. when I launch the msi, application launches without elevation. but through exe bundle package it launches with elevation only,

please help me with this issue.

**Comment by robmen** [comment]
Launch the application from the Bundle instead. If your Bundle is per-machine, then all MSI packages are executed elevated.
